oat-sa/extension-tao-itemqti
# Filter out interactions from the panel based on their visibility

Related https://oat-sa.atlassian.net/browse/AUT-1980

 - check if each interaction is visible in the authoring using the key pattern `taoQtiItem/creator/interaction/<interaction>`
 - same for PCI using `taoQtiItem/creator/customInteraction/<typeIdentifier>`
 - A small util has been added to check if a qti element is visible (and cleanup the helper)


### How to test:
 - run the unit test : `cd tao/views/build && npm ci && npx grunt connect:test taoqtiitemtest`
 - configure some interactions to be hidden in `config/tao/client_lib_config_registry.conf.php`, for example
```
                'taoQtiItem/creator/interaction/endAttempt' => 'hide',
                'taoQtiItem/creator/customInteraction/likertScale' => 'hide',
``` 
  and open the authoring to check if the interactions aren't available in the sidebar.
